### ISSUE TITLE
Error using h2o.randomForest

### ISSUE THREAD

**Issue by tressoldi** [h2oai/h2o-tutorials#15]
With this syntax:
>data=h2o.uploadFile("RIC2Simone.csv", header = TRUE,sep=";", 
                        destination_frame = "data")
>data[,1] = as.factor(data[,1])

>splits <- h2o.splitFrame(data = data, 
                         ratios = c(0.50, 0.25),  #partition data into 50%, 25%, 25% chunks
                         seed = 1)  #setting a seed will guarantee reproducibility
>train <- h2o.assign(splits[[1]], "train.hex")
>valid <- h2o.assign(splits[[2]], "valid.hex")
>test <- h2o.assign(splits[[3]],"test.hex")

>response <- "Stimolo"

>predictors<-setdiff(names(data),response)

>m1 <- h2o.deeplearning(
  model_id="dl_model_first", 
  training_frame=train, 
  validation_frame=valid,   
  x=predictors,
  y=response,
  hidden=c(32,32,32),            
  epochs=1000000,              
  score_validation_samples=10000,   
  stopping_rounds=2,
  stopping_metric="misclassification", 
  stopping_tolerance=0.01,
  nfolds=5,
  fold_assignment="Stratified", 
  variable_importances=T    
)

I got the following error message:

Illegal argument(s) for DRF model: DRF_model_R_1487947076967_1.  Details: ERRR on field: _train: Training data must have at least 2 features (incl. response).

**Comment by ledell** [comment]
This is not the right place to file bug reports for H2O.  This repository is for issues related to h2o-tutorials only.  Please post to [h2ostream](https://groups.google.com/forum/#!forum/h2ostream) or [StackOverflow](http://stackoverflow.com/questions/tagged/h2o).  That being said, it looks like H2O seems to think that you only have one column in your training data, so you should double check that your `predictors` and `response` variables are valid and that your `data` object was imported correctly.
